memgpt/local_llm/lmstudio/settings.py

```diff
@@ -9,5 +9,8 @@
         # '\n#',
         # '\n\n\n',
     ],
-    "max_tokens": 500,
+    "max_tokens": 3072,
+    "lmstudio": {"context_overflow_policy": 2},
```

Comment:
Ideally we shouldn't really be using any context overflow policies, since MemGPT has its own way of handling this and this might conflict. But if this works as a bandaid fix for some other bug wrt open LLM integration I'm OK with leaving it for now.

Reply:
Ah ok. Well it may work ok without it. As you say, I think its the endpoint change that's doing the heavy lifting for... reasons lol.